distribution/docs/src/main/resources/_contents/_catalog-contents/extending-catalog-contents.adoc

```diff
@@ -1231,6 +1232,39 @@ Use this pre-ingest plugin as a convenience to apply basic rules for your metaca
 
 None
 
+===== Metacard Validity Marker
+
+The Metacard Validity Marker Pre-Ingest plugin makes modifications to `CreateRequest` and `UpdateRequest` metacards.
+
+The plugin runs the metacards contained in the `CreateRequest` and `UpdateRequest` with each service registered as a `MetacardValidator`.
```

Comment:
Maybe "The plugin runs each metacard in the request against each registered `MetacardValidator` service"?